deployments/common/image/common-scripts/jupyter-lab-extension-setup

```diff
@@ -1,30 +1,14 @@
 #! /bin/bash
 
-set -eu
+set -eux
 
 jupyter nbextension enable --py --sys-prefix  ipyvuetify
 jupyter nbextension enable --py --sys-prefix  ipysplitpanes
 
-jupyter labextension install \
```

Comment:
Did you discover that installing packages this way was causing problems?  (versus npm installs)

Reply:
I think those are all obsolete now due to JL  packaging improvements.   So e.g. for ipyvuetify the install instructions say:
`
$ pip install ipyvuetify
$ jupyter nbextension enable --py --sys-prefix ipyvuetify

# for Jupyter Lab < 3
$ jupyter labextension install jupyter-vuetify
`
and we are using jupyterlab 3+.   I think these days all the JL stuff happens in the pip install.

Likewise,  for the npm installs,  since the old installs did not say -g (global),  they were being installed in $HOME and not even visible on our hubs.   So I think they can all be removed,  although pip may be duplicating that work and will nominally still be there.

Reply:
Ah, thanks for that explanation, makes good sense.